ccm_web/server/src/app.module.ts

```diff
@@ -39,9 +42,20 @@ const logger = baseLogger.child({ filePath: __filename})
         define: { underscored: true } // Included here to ensure session table uses snake_case
       })
     }),
-    UserModule,
-    CanvasModule
+    CanvasModule,
+    APIModule
   ],
   providers: [UserService]
 })
-export class AppModule {}
+export class AppModule implements NestModule {
+  configure (consumer: MiddlewareConsumer): void {
+    consumer.apply(helmet())
+    // Exclude ltijs routes, which are already protected by helmet
+      .exclude(
+        { path: '/lti', method: RequestMethod.POST },
+        { path: '/lti', method: RequestMethod.GET },
+        '/lti/(.*)'
+      )
+      .forRoutes('*')
+  }
+}
```

Comment:
I just added this, which should ensure all route are covered by `helmet` without providing redundancy. It made sense to do it in this PR since the decoupling caused the lack of protection. I will link the other issue to this PR as well.

Reply:
You can see which routes this is being applied to by replacing the `.apply(helmet()` with something like the following:
```
      .apply((req: Request, res: Response, next: NextFunction) => {
        console.log(req.method)
        console.log(req.baseUrl)
        helmet()(req, res, next)
      })

```